app/konnect/service-catalog/index.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
This looks like a net new file! Just out of curiosity, what's this meant to be?

Reply:
This is the overview page that exists now: 
https://docs.konghq.com/konnect/servicehub/

but I renamed the directory the file lives in so the URL would be `/service-catalog/` instead of `/servicehub/`

Reply:
Looks like you might've forgotten to delete the old file, so it looks like there's two of them now.